team/daniel-howard.yaml

```diff
@@ -0,0 +1,27 @@
+# Replace CAPITALIZED placeholders with your info
+- title: Daniel Howard
+  avatar: https://avatars.githubusercontent.com/dphow
+  role: HPC Consultant
+  organizations:
+    - name: NSF National Center for Atmospheric Research
+  bio: >
+    Daniel is an HPC Consultant at National Science Foundation National Center for Atmospheric Research (NSF NCAR), within the 
+    Computational and Information Systems Lab (CISL). He primarily supports users on the Derecho and Casper supercomputing clusters
+    towards running large scale Earth system science applications. Daniel primarily contributes towards projects to facilitate 
+    their readiness and performance with GPU computing, supporting porting efforts and performance profiling. Daniel also has recently 
+    studied under Colorado University - Boulder's Earth Data Analytics program and regularly engages with NSF NCAR's Earth System Data Science 
+    (ESDS) initiative, coordinating and supporting various hackathon events for the scientific community.
+  expertise:
+    - High Performance Computing
+    - GPU Programming - OpenACC, OpenMP, and CUDA
+    - Performance Profiling and Optimization
+    - Remote Sensing workflows, ie Geopandas and Rasterio with Python
```

Comment:
Just noticed this
```
..., ie Geopandas
```
Should that be `in` or `i.e.`?

Reply:
i'm going to merge, feel free to modify in a follow up!